apps/finance/contracts/Finance.sol

```diff
@@ -166,32 +160,35 @@ contract Finance is AragonApp, ERC677Receiver {
         external
```

Comment:
I would remove this function entirely now that we have the new vault

Reply:
:+1: